sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/DynamicMethodInfo.scala

```diff
@@ -110,7 +127,7 @@ object DynamicMethodInfo {
       val fieldNumber = fieldIdx + 2
       fieldDescriptor.setNumber(fieldNumber)
       fieldDescriptor.setType(mapJavaTypeToProtobuf(paramType))
-      // todo entity key
+      addEntityKeyIfNeeded(paramName, fieldDescriptor)
```

Comment:
Only adding keys for path params.

We need to decide about where we can add `@EntityKey`. I will open an issue to discuss it.